app/controllers/user.js

```diff
@@ -14,7 +21,26 @@ exports.createUser = async (req, res, next) => {
     const password = await encriptPassword(req.body.password);
     const user = await registerUser(name, lastName, email, password);
     logger.info('user created');
-    return res.status(201).json({ name: user.name, lastName: user.lastName, email: user.email });
+    return res.status(201).json({
+      name: user.name,
+      lastName: user.lastName,
+      email: user.email
+    });
+  } catch (error) {
+    logger.error(error.message);
+    return next(error);
+  }
+};
+
+exports.signIn = async (req, res, next) => {
+  try {
+    logger.info('Do signIn');
+    const { email, password } = req.body;
+    const user = await findUserByEmail(email);
+    if (!user) throw userFindError(USER_FIND_ERROR);
```

Comment:
For when you do not find a user with the email provided or when the password does not match, you should return the same error message, it is for security, so we do not give clues of which of the two is right or wrong.